src/PresentationalComponents/Breadcrumbs/Breadcrumbs.js

```diff
@@ -41,7 +44,7 @@ const Breadcrumbs = ({ current }) => {
       if (location[2] === 'pathways') {
         crumbs = [
           {
-            title: 'Advisor pathways',
+            title: 'Pathways',
             navigate: '/recommendations/pathways',
           },
```

Comment:
I am not really sure this works as expected: try to navigate to any pathways details page and you will see Recommendations as the first breadcrumb. Shouldn't it be Pathways instead and lead to /recommendations/pathways instead of /recommendations?

Reply:
This issue is fixed in the second commit I added 👍🏼